examples/scaling-criteo/03-Training-with-Merlin-Models.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>Maybe after this sentence "We will use the same techniques to train a deep learning model for the <a href="https://ailab.criteo.com/download-criteo-1tb-click-logs-dataset/" target="_blank">Criteo 1TB Click Logs dataset</a>." You can mention how many days of criteo dataset you are going to use for training, and evaluation.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>